src/main.rs

```diff
@@ -202,6 +202,7 @@ After={service}
 
 [Swap]
 What={root}dev/{zram_device}
+Options=pri=100
```

Comment:
Please use `Priority=100` instead.

Reply:
I now see https://github.com/systemd/zram-generator/pull/12#issuecomment-582982875. So maybe let's leave this with Options= for now, and figure out why Priority does now work later.

Reply:
I'll test this on my laptop today and check what's the problem.